tests/layer_tests/tensorflow_tests/test_tf_ExpandDims.py

```diff
@@ -40,3 +41,53 @@ def test_expand_dims_basic(self, params, ie_device, precision, ir_version, temp_
         self._test(*self.create_expand_dims_net(**params),
                    ie_device, precision, ir_version, temp_dir=temp_dir,
                    use_legacy_frontend=use_legacy_frontend)
+
+
+class TestExpandDimsComplex(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        # generate elements so that the input tensor may contain repeating elements
+        assert 'param_real:0' in inputs_info
+        assert 'param_imag:0' in inputs_info
+
+        input_shape = inputs_info['param_real:0']
+
+        inputs_data = {}
+        inputs_data['param_real:0'] = rng.integers(-10.0, 10.0, input_shape).astype(np.float32)
+        inputs_data['param_imag:0'] = rng.integers(-10.0, 10.0, input_shape).astype(np.float32)
+
+        return inputs_data
+
+    def create_expand_dims_complex_net(self, input_shape, axis):
+        tf.compat.v1.reset_default_graph()
+        with tf.compat.v1.Session() as sess:
+            param_real = tf.compat.v1.placeholder(np.float32, input_shape, 'param_real')
+            param_imag = tf.compat.v1.placeholder(np.float32, input_shape, 'param_imag')
+
+            complex = tf.raw_ops.Complex(real=param_real, imag=param_imag)
+
+            axis = tf.constant(axis, dtype=tf.int32)
+
+            result = tf.raw_ops.ExpandDims(input=complex, axis=axis)
+
+            tf.raw_ops.Real(input=result)
+            tf.raw_ops.Imag(input=result)
+
+            tf.compat.v1.global_variables_initializer()
+            tf_net = sess.graph_def
+
+        return tf_net, None
+
+    test_basic = [
+        dict(input_shape=[], axis=0),
+        dict(input_shape=[2, 3], axis=1),
+        dict(input_shape=[2, 3, 4], axis=-1),
+        dict(input_shape=[2, 6, 5], axis=-2),
+    ]
+
```

Comment:
please check different `input` types and `axis` types.

Reply:
for input dtype it can either be `float32` or `float64`. However I'm getting this error for `float64` 

```
E       tensorflow.python.framework.errors_impl.InvalidArgumentError: Graph execution error:
E
E       Detected at node 'Complex' defined at (most recent call last):
E       Node: 'Complex'
E       No OpKernel was registered to support Op 'Complex' used by {{node Complex}} with these attrs: [Tout=DT_COMPLEX64, T=DT_DOUBLE]
E       Registered devices: [CPU]
E       Registered kernels:
E         device='XLA_CPU_JIT'; Tout in [DT_COMPLEX64, DT_COMPLEX128]; T in [DT_FLOAT, DT_DOUBLE]
E         device='GPU'; T in [DT_DOUBLE]; Tout in [DT_COMPLEX128]
E         device='GPU'; T in [DT_FLOAT]; Tout in [DT_COMPLEX64]
E         device='CPU'; T in [DT_DOUBLE]; Tout in [DT_COMPLEX128]
E         device='CPU'; T in [DT_FLOAT]; Tout in [DT_COMPLEX64]
E         device='XLA_GPU_JIT'; Tout in [DT_COMPLEX64, DT_COMPLEX128]; T in [DT_FLOAT, DT_DOUBLE]
E
E                [[Complex]]

../../../.pyenv/versions/3.10.7/envs/ov-py310/lib/python3.10/site-packages/tensorflow/python/client/session.py:1420: InvalidArgumentError
```

Reply:
> for input dtype it can either be `float32` or `float64`. However I'm getting this error for `float64`

I tried to change dtype in other translator's unit tests to `float64` and there is the same error so I guess it's not related to my translator.